matron/src/event_types.h

```diff
@@ -3,6 +3,7 @@
 #include "oracle.h"
 #include "osc.h"
 #include <stdint.h>
+#include <lua.h>
```

Comment:
ouch! it hurts! but ok.

Reply:
the only other option that i can think of is to explore keeping the ops structure opaque and move it to a separate header. that might be enough to keep the exposure to lua limited to both weaver.c and the external code.

Reply:
awareness of lua has been moved to `event_custom.h` which is only needs to be included by external code which implements  custom events and internally to `event.c` and `weaver.c`.